pom.xml

```diff
@@ -89,17 +89,17 @@
 		<providermanagementVersion>2.10.0</providermanagementVersion>
 		<referenceapplicationVersion>2.10.0-SNAPSHOT</referenceapplicationVersion>
 		<referencemetadataVersion>2.10.1</referencemetadataVersion>
-		<registrationappVersion>1.15.0-SNAPSHOT</registrationappVersion>
+		<registrationappVersion>1.15.0</registrationappVersion>
 		<registrationcoreVersion>1.9.0</registrationcoreVersion>
 		<reportingRestVersion>1.11.0</reportingRestVersion>
 		<reportingVersion>1.20.0-SNAPSHOT</reportingVersion>
 		<reportingCompatibilityVersion>2.0.6</reportingCompatibilityVersion>
 		<reportingUIVersion>1.6.0</reportingUIVersion>
 		<serializationxstreamVersion>0.2.14</serializationxstreamVersion>
-		<uicommonsVersion>2.11.0-SNAPSHOT</uicommonsVersion>
+		<uicommonsVersion>2.11.0</uicommonsVersion>
 		<uiframeworkVersion>3.16.0</uiframeworkVersion>
 		<uilibraryVersion>2.0.6</uilibraryVersion>
-		<ugandaemrfingerprintVersion>1.0.10</ugandaemrfingerprintVersion>
+		<ugandaemrfingerprintVersion>1.0.11-SNAPSHOT</ugandaemrfingerprintVersion>
```

Comment:
@slubwama What is in this snapshot? Do I need to release?

Reply:
This Pr is still pending https://github.com/METS-Programme/openmrs-module-ugandaemr-fingerprint/pull/38